### PR TITLE
Fix NFT mint process

### PR DESCRIPTION
**Pull request by nomadbitcoin**
## Brief Information

The type of this pull request is:

- [x] bug fixing
- [ ] new feature
- [ ] others

### What does this PR do?
1. **Language Sections Lesson Checks**: Improved the logic for checking lessons within different language sections and updated the associated queries for better accuracy.
2. **Email Copy Update for NFT Campaign**: Revised the email copy used in our NFT campaign to ensure clearer communication and better engagement with recipients.

### CODE REVIEW

File: functions/email_templates/nft_delivery.js [hunk 1]
```diff
@@ -1,19 +1,23 @@
 function template(params) {
   // course_title, wallet_address, nft_contract, nft_id) {
   return `
-Fala builder 👷👷‍♀️!<br>
+Hello builder 👷👷‍♀️!<br>
```
Comment (danicuki) [inline functions/email_templates/nft_delivery.js:4]: Nós estávamos deixando esses emails transacionais nas duas línguas (pt e en)

File: functions/lib/checkUserLessons.js [hunk 2]
```diff
@@ -6,26 +6,25 @@ async function userCompletedCourse(userId, courseId) {
...
         .then(async (courses) => {
           const course = courses.data()
-          return Object.keys(course.sections)
-            .map((section) => {
-              return course.sections[section].map((lesson) => {
-                return lesson.file
-              })
-            })
-            .flat()
+          // Find the first language that has sections
```
Comment (danicuki) [inline functions/lib/checkUserLessons.js:16]: this will break for existing courses unless you add `metadata` field and sections in all of them

Comment (nomadbitcoin) [reply functions/lib/checkUserLessons.js:16]: All current courses have been updated and now have the appropriate metadata fields

File: functions/email_templates/nft_delivery.js (hunk 1 above)
Comment (nomadbitcoin) [reply functions/email_templates/nft_delivery.js:6]: you are right, I just updated

**Review by danicuki — APPROVED** [review]

**Pull request merged**